examples/run-blockchain/index.js

```diff
@@ -17,14 +17,24 @@ var blockchainDB = levelMem()
 
 var state = new Trie()
 
-var blockchain = new Blockchain(blockchainDB)
-blockchain.ethash.cacheDB = level('./.cachedb')
+var validate = false
+
+var hardfork = 'byzantium'
```

Comment:
This data is present in `test-data.json`. Can you get it from there? this way it will be up-to-date if the data is changed. Something like this would do it: `var hardfork = testData.network.toLowerCase()`

Reply:
Good point. I'll read it from the testData.

Reply:
done